### PR TITLE
Don't move the plus when adding services.

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- add a service
- note the plus is in the upper left of the canvas
- add another service
- the plus should not move

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4175/
Test PASSed.

File: jujugui/static/gui/src/test/test_environment_view.js [hunk 1]
```diff
@@ -328,7 +328,7 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
       });
     });
 
-    it('should relocate the plus nearby', function() {
+    it.skip('should relocate the plus nearby', function() {
```
Comment (hatched) [inline jujugui/static/gui/src/test/test_environment_view.js]: might as well just remove this test

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4182/
Test PASSed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4184/
Test PASSed.

**Comment by hatched** [comment]
:+1: :ok: :shipit: 
Thanks!

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**